build.sbt

```diff
@@ -121,6 +121,7 @@ lazy val bench = project
   .settings(
     name := "bench",
     coverageEnabled := false,
+    scalacOptions += "-Wconf:cat=unused-nowarn:s",
```

Comment:
suppresses this only for `bench` project

```
 [error] /home/runner/work/cats-parse/cats-parse/bench/src/main/scala/cats/parse/bench/fastparse.scala:50:43: @nowarn annotation does not suppress any warnings
[error]     P(space ~ "\"" ~/ (strChars | escape).rep.! ~ "\"").map(JString.apply)
[error]                                           ^
[error] one error found
```

Reply:
Hi @johnynek any concerns about this?

Reply:
No concern for bench.

Reply:
Thanks